src/components/Menu.scss

```diff
@@ -0,0 +1,73 @@
+.Menu{
```

Comment:
I would recommend using [kebab-case](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841) both for `scss` filenames and class names.

It's a common convention to use this casing and makes the code homogeneous and cleaner